example-apps/react-js/src/pages/Home/index.jsx

```
@@ -4,15 +4,24 @@ import GlassCard from "@components/GlassCard";
 import Tooltip from "@components/Tooltip";
 import styles from "./Home.module.css";
 import HomeHelmet from "./HomeHelmet";
+import { useEffect } from "react";
```

ESLint

[error] 'useEffect' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)